docs/pipeline-components-and-applications/loaders-storage-targets/bigquery-loader/previous-versions/bigquery-loader-1.x/_diagram.md

```diff
@@ -0,0 +1,24 @@
+At the high level, BigQuery loader reads enriched Snowplow events in real time and loads them in BigQuery using the Storage Write API.
```

Comment:
Doesn’t 1.x use Streaming Inserts API rather than Storage Write API?

Reply:
Yeah...something like this. I'm confused by the naming. In docs I think they call it [legacy streaming API](https://cloud.google.com/bigquery/docs/streaming-data-into-bigquery). But the new one is also streaming, right?

Reply:
Let's be consistent with the GCP docs.  Change this sentence to say "legacy streaming API" and you can even link to that page you just shared.  And for the new one let's call it "storage write API" and [link to this](https://cloud.google.com/bigquery/docs/write-api).